config/site/src/_config.yaml

```diff
@@ -7,6 +7,8 @@ github_url: https://github.com/block/elasticgraph
 x_url: https://x.com/elasticgraph
 x_username: "@ElasticGraph"
 support_email: elasticgraph@squareup.com
+description: ElasticGraph is an open-source framework for indexing, searching, grouping, and aggregating data
```

Comment:
I was thinking of `Schema-driven, scalable, cloud-native, batteries-included GraphQL with superpowers` as a replacement for this. Do you think there's value in keeping both?

Reply:
This is for the HTML `meta name=description` tag (SEO) and the subtitle on the homepage. We could rename the variable to `seo_description`.